app/controllers/stats_controller.rb

```diff
@@ -1,3 +1,5 @@
+require 'csv'
+
 class StatsController < ApplicationController
   # @note (Mohammed Seedat): Interim solution to allow 'Closed Cases'
```

Comment:
General comment about this controller is that it is getting quite large - I think there may be an argument to split things out into service classes that could be more easily unit tested. I think we could discuss it as something to do in a separate PR though.